cli/bench/space.go

```diff
@@ -54,3 +71,23 @@ func dropBenchmarkSpace(tarantoolConnection *tarantool.Connection) error {
 	}
 	return nil
 }
+
+// fillBenchmarkSpace fills benchmark space with a PreFillingCount number of records using connectionPool for fast filling
+func fillBenchmarkSpace(ctx context.BenchCtx, connectionPool []*tarantool.Connection) {
+	var waitGroup sync.WaitGroup
+	filledCount := 0
+	for i := 0; i < ctx.Connections; i++ {
+		waitGroup.Add(1)
+		go func(tarantoolConnection *tarantool.Connection) {
+			defer waitGroup.Done()
+			for filledCount < ctx.PreFillingCount {
+				tarantoolConnection.Exec(tarantool.Insert(
+					benchSpaceName,
+					[]interface{}{common.RandomString(ctx.KeySize), common.RandomString(ctx.DataSize)}))
```

Comment:
The string length is greater than 100. Why this is bad described in the previous review.